services/api/service.go

```diff
@@ -1512,7 +1512,7 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		// Without cancellations, discard lower or similar value submissions to previous top bid
 		if !isCancellationEnabled && payload.Value().Cmp(topBidValue) < 1 {
 			log.Info("rejecting submission because it is lower or equal to the top bid (redis)")
-			w.WriteHeader(http.StatusOK)
+			api.Respond(w, http.StatusAccepted, "ignoring submission because it is lower or equal to the top bid and cancellations are not enabled")
```

Comment:
Trying to shorten the response...

```suggestion
			api.Respond(w, http.StatusAccepted, "ignoring submission because not highest value")
```

Reply:
Or should we introduce a method `api.RespondMessage` to have a unified message response structure?

Reply:
nice! edited